src/internal/rates/index.ts

```diff
@@ -0,0 +1,90 @@
+import { Provider } from '@ethersproject/abstract-provider';
+import { Signer } from '@ethersproject/abstract-signer';
+import { Protocols } from '../../types/index.js';
+import { DEFAULT_RATES_CONFIG, RatesConfig } from './config.js';
+import { exchangeRateAave, exchangeRateCERC20, exchangeRateERC4626, exchangeRateEuler, exchangeRateYearn, interestRateAave, interestRateCERC20, interestRateERC4626, interestRateEuler, interestRateYearn } from './protocols/index.js';
+
+/**
+ * Retrieve the exchange rate for a lending protocol and cToken/pool.
+ *
+ * @param p - protocol enum value of the lending protocol associated with a swivel market
+ * @param a - address of the market's cToken
+ * @param s - ethers provider or signer
+ * @param c - optional rates configuration object
+ * @returns the exchange rate of the protocol's cToken/pool to underlying (the scale of the value depends on the protocol)
+ */
+export async function getExchangeRate (p: Protocols, a: string, s: Provider | Signer, c?: RatesConfig): Promise<string | undefined> {
```

Comment:
`get` is a garbage word. Use it if you must, if it suits some larger pattern you have going in here, or avoids a clash, but if not `exchangeRate` > `getExchangeRate` 

as always lol...

Reply:
i do agree, the reason i named it like this, is the following:

```ts
import { exchangeRate } from './internal/rates/index.js';

// ERROR - you're redefining the name of the import... 
// ...very likely to have clashing variable names
const exchangeRate = exchangeRate(...);
```
you'd either have to name your local variables differently (which sucks) or you have to alias your imports (which also sucks)...
```ts
import { exchangeRate as getExchangeRate } from './internal/rates/index.js';

const exchangeRate = getExchangeRate(...);
```